config/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
-  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
-  "https://checkstyle.org/dtds/configuration_1_3.dtd">
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
```

Comment:
Is there a reason you used this one?  Think we should be using a standard Checkstyle from the programme like the one in spring boot template or one we use in our existing apps?